contracts/LpSugar.vy

```diff
@@ -214,88 +197,11 @@ def _pools(with_convertor: bool) -> DynArray[address[3], MAX_POOLS]:
 
   return pools
 
-@external
-@view
-def toMigrate(_account: address) -> DynArray[Lp, MAX_POOLS]:
```

Comment:
@ethzoomer this is still used by the dapp, I'm down to remove the migration, but will need it cleaned up from the `app` repo as well

Reply:
Ty ty, this is good to merge otherwise.